src/app/variables.css

```diff
@@ -107,6 +107,8 @@ or "warn/action" ineastd of "red/green"
   --color-burnt-sienna: #ec6868;
   --color-ufo-green: #00a04b;
   --color-deep-green: #00d563;
+  --color-ufo-green: #2bd67b;
```

Comment:
Did you mean to name this something else? there is a `--color-ufo-green` already.